model/task/task.go

```diff
@@ -1412,7 +1418,7 @@ func ActivateTasksByIdsWithDependencies(ids []string, caller string) error {
 		return errors.Wrap(err, "getting recursive dependencies")
 	}
 
-	if err = ActivateTasks(append(tasks, dependOn...), time.Now(), caller); err != nil {
+	if err = ActivateTasks(append(tasks, dependOn...), time.Now(), true, caller); err != nil {
```

Comment:
Should we put a bool parameter in this function? In case we want to activate tasks by ids without dependencies at some point